SDKTest.js

```diff
@@ -48,54 +50,43 @@ describe('safientMain', async () => {
       // SUCCESS : create a safe
       await sc.safientMain.createSafe(
         beneficiaryAddress, // 2nd account
-        safeIdOnThreadDB,
+        safeId1OnThreadDB,
+        1,
```

Comment:
Create a map to the claim type in the tests too. It's much better to import the claim types through SDK.